docs/advanced/components/components.md

```diff
@@ -72,13 +72,16 @@ created () {
 
 ### [Vuex](/stores/vuex/vuex/)
 
-Using Vuex you're implementing the Flux architecture
+Using Vuex you're implementing the Flux architecture.
 
 ### Dependency Injection
 
-Allows you to define a service via the `provide` property, which should be an object or a function that returns an object, and make it available to all of the component's descendants, not juts it's direct children.
+There are two alternative Vue.js handles dependency injection modules:
```

Comment:
This should be, in similar words:
_There are two alternatives on how Vue.js handles dependency injection modules:_ 
as _There are two alternative Vue.js handles dependency injection modules_ is missing some sense IMO.